### PR TITLE
New src directory with .java

### PR DESCRIPTION
**Pull request by leonnahart**
Created a new src directory and moved all .java files

### CODE REVIEW

File: src/Food.java [hunk 1]
```diff
@@ -1,3 +1,4 @@
+package src;
```
Comment (mangkoran) [inline src/Food.java:1]: Here

File: src/Fries.java [hunk 2]
```diff
@@ -1,3 +1,4 @@
+package src;
```
Comment (mangkoran) [inline src/Fries.java:1]: Here

File: src/Popcorn.java [hunk 3]
```diff
@@ -1,3 +1,4 @@
+package src;
```
Comment (mangkoran) [inline src/Popcorn.java:1]: Here

File: src/Water.java [hunk 4]
```diff
@@ -1,3 +1,4 @@
+package src;
```
Comment (mangkoran) [inline src/Water.java:1]: Here

**Review by mangkoran — COMMENTED** [review]
I don't really know about package, but should these lines exist?

**Comment by mangkoran** [comment]
If we want to utilize package, `Food` related class should be put within `src/food` directory

**Comment by leonnahart** [comment]
It appears automatically after I moved all the files into the directory

**Comment by mangkoran** [comment]
Ok then, I think we should use package. Could you please move the related `.java` files to `src/food` directory? It will look more or less like this:
```
└── src
    └── food
        ├── Food.java
        └── etc..
```

**Comment by leonnahart** [comment]
Alright, will do

**Comment by leonnahart** [comment]
I have made the changes, @mangkoran please review it. Thank you

**Review by mangkoran — APPROVED** [review]

**Comment by mangkoran** [comment]
Alright, thank you too!

**Pull request merged**